scripts/build.py

```diff
@@ -156,29 +222,99 @@ def process_js(source_path, target_path):
     _minify(source_path, target_path)
 
 
-def build_minified_third_party_libs(output_directory):
+def get_dependency_filepaths(dependency, filepaths):
+    """Extracts dependency filepaths into filepaths object.
+
+    Args:
+        source_paths: dict(str, str). Dependency information object.
+        filepaths: dict(str, list(str)). Filepaths object with three lists each
+            for different file type (js, css, fonts).
+    """
+    if "targetDir" in dependency:
```

Comment:
Use single quotes in python, here and elsewhere

Reply:
Done.